packages/config/config/devices/0x015f/mh-s220.json

```diff
@@ -0,0 +1,177 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH-S220",
+	"description": "Micro Switch",
+	"devices": [
+		{
+			"productType": "0x2201",
+			"productId": "0x1252"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Switch 1",
+			"maxNodes": 5
+		},
+		"2": {
+			"label": "Switch 2",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/templates/master_template.json#state_after_power_failure_off_prev"
+		},
+		{
+			"#": "2",
+			"label": "External Switch Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary Buttons",
+					"value": 0
+				},
+				{
+					"label": "2-State Switches",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "2-State Switches",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Toggle Switch without specified on/off position",
+					"value": 0
+				},
+				{
+					"label": "Toggle Switch with specified on/off position",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Switch 1 - Operation Report to Association Group 1",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Report both on/off",
+					"value": 0
+				},
+				{
+					"label": "Report on only",
+					"value": 1
+				},
+				{
+					"label": "Report off only",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Switch 1 - Off Timer",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"defaultValue": 1
+		},
+		{
+			"#": "6",
+			"label": "Switch 1 - Off Timer Interval",
+			"unit": "100ms",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true
+		},
+		{
+			"#": "7",
+			"label": "Switch 1 - Operation Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Switch Immediately",
+					"value": 0
+				},
+				{
+					"label": "Zero Crossing Switch",
```

Comment:
Is there a description for this parameter? I don't think its immediately clear what this does.

Reply:
The manual doesn't say anything more than this, however I have added a description based on other relay devices that I've got that have zero-crossing detection in them.